src/Apps/Settings/Routes/Orders/Components/__tests__/SettingsOrders.jest.tsx

```diff
@@ -0,0 +1,96 @@
+import { screen } from "@testing-library/react"
+import { SettingsOrdersFragmentContainer } from "Apps/Settings/Routes/Orders/Components/SettingsOrders"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapperTL"
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+jest.mock("Components/Pagination/CommercePagination", () => ({
+  CommercePaginationFragmentContainer: () => null,
+}))
+
+// Mock SettingsOrdersRowLoader since it uses useLazyLoadQuery which is difficult to test with Suspense
+jest.mock("../SettingsOrdersRowLoader", () => ({
+  SettingsOrdersRowLoader: () => {
+    return null // Return null to simplify testing - we test SettingsOrdersRow separately
+  },
+}))
+
+const { renderWithRelay } = setupTestWrapperTL({
+  Component: SettingsOrdersFragmentContainer,
+  query: graphql`
+    query SettingsOrders_Test_Query @relay_test_operation {
+      me {
+        ...SettingsOrders_me
+      }
+    }
+  `,
+})
+
+describe("SettingsOrders", () => {
+  it("renders orders list when orders exist", () => {
+    renderWithRelay({
+      Me: () => ({
+        orders: {
+          edges: [
+            {
+              node: {
+                internalID: "order-1",
+              },
+            },
+            {
+              node: {
+                internalID: "order-2",
+              },
+            },
+          ],
+        },
+      }),
+    })
+
+    // Component should render without errors when orders exist
+    expect(document.body).toBeInTheDocument()
+  })
+
+  it("renders empty state when no orders exist", () => {
+    renderWithRelay({
+      Me: () => ({
+        orders: {
+          edges: [],
+        },
+      }),
+    })
+
+    expect(
+      screen.getByText("Orders can not be loaded at this time."),
```

Comment:
I have a [similar question](https://github.com/artsy/force/pull/16202#discussion_r2427029454) about the messaging for the empty state. This language sounds an error state, rather than an [empty state messaging](https://github.com/artsy/force/blob/3308081660d75f50e44ad1af5dbc7a680b409397/src/Apps/Settings/Routes/Orders/Components/SettingsOrders.tsx#L69).